.github/workflows/bicep_deploy.yml

```diff
@@ -0,0 +1,137 @@
+name: Deploy via Bicep
+
+# TODO:FN-72 consider concurrency
```

Comment:
I'll address this shortly, but might need to think a bit more when we actually have multiple environments controlled by Bicep. Note that `bicep_workflow.yml` has a concurrency limiter anyway.